src/partials/70_section-blog-posts.html

```diff
@@ -16,59 +16,68 @@
     <div class="row">
       <div class="col-4">
         <div class="post-box">
-          <div class="cover">
-            <div class="details">
-              <div class="date"><i class="fas fa-calendar-alt"></i> 15 Jan 2016</div>
-              <div class="comments"><i class="fas fa-comments"></i> 4 Comments</div>
+          <a href="#">
```

Comment:
Element **a** zawierający cover'a jest nadmiarowy. Widać to szczególnie na kolejnych przykładach, gdzie mamy **a** zawierający child'a **a** (Read more). Generalnie jak coś jest **a** to nie może zawierać klikalnych childów.

Usuń **a** wyższego poziomu.